scopes/component/isolator/isolator.main.runtime.ts

```diff
@@ -269,7 +273,8 @@ export class IsolatorMain {
     await this.updateWithCurrentPackageJsonData(capsulesWithPackagesData, capsuleList);
     const installOptions = Object.assign({}, DEFAULT_ISOLATE_INSTALL_OPTIONS, opts.installOptions || {});
     if (installOptions.installPackages) {
-      await this.installInCapsules(capsulesDir, capsuleList, installOptions, opts.cachePackagesOnCapsulesRoot ?? false);
+      const rootDir = opts.installOptions?.useNesting ? capsuleList[0].path : capsulesDir;
```

Comment:
I don't understand this line.
What `capsuleList[0].path` is represents here?
can you write a little explanation (better a comment in the code as well)